### DISCUSSION TITLE
Support fzf syntax in pickers

### DISCUSSION THREAD

**Discussion by rcorre** [helix-editor/helix#5838] [Ideas] [answered]
Could we add support for [fzf syntax](https://github.com/junegunn/fzf#search-syntax) in pickers?

| Token     | Match type                 | Description                          |
| --------- | -------------------------- | ------------------------------------ |
| `sbtrkt`  | fuzzy-match                | Items that match `sbtrkt`            |
| `'wild`   | exact-match (quoted)       | Items that include `wild`            |
| `^music`  | prefix-exact-match         | Items that start with `music`        |
| `.mp3$`   | suffix-exact-match         | Items that end with `.mp3`           |
| `!fire`   | inverse-exact-match        | Items that do not include `fire`     |
| `!^music` | inverse-prefix-exact-match | Items that do not start with `music` |
| `!.mp3$`  | inverse-suffix-exact-match | Items that do not end with `.mp3`    |

Right now, the picker appears to interpret these characters literally.
I mainly find myself missing "inverse" and "exact".

**Comment by pascalkuthe** [comment, marked as answer]
Already done in #5114

**Reply by rcorre** [reply]
That's great, thanks! Sorry, I did try a number of searches but didn't see it.
